ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -114,12 +179,164 @@ class LargeTitleView: UIView {
     private var smallMorphingAvatarViewAnimator: UIViewPropertyAnimator? // responsible for animating the transform of smallMorphingAvatarView
     private var smallAnimatorRunningObserver: NSKeyValueObservation? // observes the running property, done to accomplish a "completion" for a pausesOnCompletion = YES animator
 
-    private let titleButton = UIButton() // button used to display the title of the current navigation item
+    private let titleAreaView = UIView(frame: .zero)
```

Comment:
i think we should try to reuse TwoLineTitleView. I think the class is intended that only one of the chevron can exists either title or subtitle. It uses MSFEasyTapButton so that the taparea target is much bigger than its icon to match apple HIG